test/includes/microcloud.sh

```diff
@@ -1050,7 +1050,7 @@ restore_system() {
 
     for n in $(seq 1 "${num_disks}") ; do
       disk="${name}-disk${n}"
-      lxc storage volume create zpool "${disk}" size=5GiB --type=block
+      lxc storage volume create zpool "${disk}" size=10GiB --type=block
```

Comment:
I'll rebase the Canonical runner PR after this got merged and see if it will cause any problems. IIRC the disks on those runners are a bit limited with 40GB (large flavor).

But as the disks aren't thick provisioned probably it will just work.